lib/src/controllers/chat/ds_file_message_bubble.controller.dart

```diff
@@ -1,71 +1,15 @@
 import 'package:filesize/filesize.dart';
-import 'package:flutter/material.dart';
 import 'package:get/get.dart';
-import 'package:path/path.dart' as path_utils;
 
 import 'package:blip_ds/src/services/ds_file.service.dart';
```

Comment:
Use the relative path instead of absolute path

Reply:
Done